app/models/test_student.rb

```diff
@@ -0,0 +1,14 @@
+# TestStudent - fake student, used for running tests on solution files
+class TestStudent < User
+  has_many :accepted_groupings, lambda {
```

Comment:
use the arrow syntax `->` instead of `lambda`

Reply:
I used the -> first but Rubocop raised an offense and suggested using lambda. Which one is recommended?

Reply:
I'm going to make a decision here and say that we should go with `->`. I'll make a separate PR that disables that cop so houndbot won't complain in the future.